M5StickC-master/src/AXP192.h

```diff
@@ -80,6 +80,7 @@ class AXP192 {
     void SetCoulombClear()  __attribute__((deprecated)); // use ClearCoulombcounter instead
```

Comment:
we could potentially use coulombcounter in future to predict battery life
but I don't know how much power does it even use